### DISCUSSION TITLE
Server connection issue

### DISCUSSION THREAD

**Discussion by barry-coleman-lp** [rOpenGov/eurostat#249] [Q&A] [answered]
Hello,

Firstly, thank you for creating this package. I am a huge fan.

I have run in to an issue today where I can no longer access the server using the R package.

I cannot find any resources online to help with this issue and I'm looking for some help.

`> get_eurostat_toc()`
`Error in open.connection(con, "rb") : cannot open the connection
In addition: Warning message:
In open.connection(con, "rb") :
  InternetOpenUrl failed: 'A connection with the server could not be established'`

**Comment by pitkant** [comment, marked as answer]
Hi, thank you for starting this discussion. I converted this into an issue so that if other people have the same issue they can find it easily. We can continue discussing the technicals there: #250
